plugins/exec-cronjob.yaml

```diff
@@ -18,11 +18,16 @@ spec:
     bin: kubectl-exec_cronjob.sh
   shortDescription: Run a CronJob immediately as Job
   description: |
+    DEPRECATION NOTICE:
```

Comment:
This might be also good for adding to CAVEATS section, as we show that post install and post upgrade.